packages/algob/src/internal/deployer.ts

```diff
@@ -1,10 +1,13 @@
 import { decode, encode } from "@msgpack/msgpack";
 import * as algosdk from "algosdk";
+import fs from "fs";
+import path from "path";
 
```

Comment:
These modules are never used. is your lint not working ?
`yarn run lint:fix`

Reply:
Seems they are not. Fixing.